src/converter/one/jfr/event/SortedEventAggregator.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package one.jfr.event;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Collections;
+
+public abstract class SortedEventAggregator extends EventAggregator {
```

Comment:
Redundant abstractions, IMO.
There are currently only two aggregators. If we go this way, we can just make MallocLeakAggregator a subclass of EventAggregator.